org.OpenT2T.Sample.SuperPopular.Lamp/Vera Dimmable Light Bulb/js/thingTranslator.js

```diff
@@ -0,0 +1,174 @@
+'use strict';
+
+var https = require('https');
+var q = require('q');
+
+// logs device state
+function logDeviceState(device) {
+    if (typeof (device) !== 'undefined') {
+        console.log('  device.name          : ' + device.name);
+        console.log('  device.props         : ' + device.props);
+    } else {
+        console.log('device is undefined');
+    }
+};
+
+// Helper method to send power state commands to Vera
+function sendPowerStateCommandToDevice(loadTarget, deviceId, relayServer, relaySessionToken, pkDevice) {
+    var deferred = q.defer();
+
+    // set up the url which needs:
+    //   the serial number of the hub
+    //   deviceid of the light bulb
+    //   target brightness
+    var path = '/relay/relay/relay/device/' + pkDevice + '/port_3480/data_request?id=action&output_format=json&DeviceNum=' + deviceId + '&serviceId=urn:upnp-org:serviceId:Dimming1&action=SetLoadLevelTarget&newLoadlevelTarget=' + loadTarget;
```

Comment:
I tried that originally and that did not work for me, can you double check your setup, maybe it depends hub to hub? or maybe I'm missing some other value.  

I'll change it to SwitchPower1 since that is more correct.